sdk/limit_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sdk
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSpanAttrValLenLimit(t *testing.T) {
```

Comment:
could these be table tests instead of individual tests? Also, a test for invalid env values falling back to the default would be good